### PR TITLE
Point to elixir-lsp, improve build process and docs

### PR DESCRIPTION
**Pull request by balduncle**
- Update JakeBecker references to point to elixir-lsp
- Update submodule to point to elixir-lsp
- Streamline build process and update README with instructions for building and installing locally
- Copy "reason for fork" text from elixir-ls README for consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+- Code formatter (requires Elixir >= 1.7. Triggered by `Alt + Shift + F` hotkey or enabling `editor.formatOnSave`)
 - Find references to functions and modules (Thanks to @mattbaker)
 - Quick symbol lookup in file (Thanks to @mattbaker)
 
-![Screenshot](https://raw.githubusercontent.com/JakeBecker/elixir-ls/master/images/screenshot.png)
+![Screenshot](https://raw.githubusercontent.com/elixir-lsp/elixir-ls/master/images/screenshot.png)
+
+## The reason for this fork
+
+This fork started when [Jake Becker's repository](https://github.com/JakeBecker/vscode-elixir-ls) became inactive for an extended period of time.  So we decided to start an active fork to merge dormant PR's and fix issues where possible. We also believe in an open and shared governance model to share the work instead of relying on one person to shoulder the whole burden.
```
Comment (balduncle) [inline README.md]: I would like to update this text across our repos to more clearly call Jake out for laying a great foundation for the community.  Later change.

File: README.md [hunk 2]
```diff
@@ -48,25 +52,17 @@ It may take some getting used to, but I highly recommend leaving `acceptSuggesti
 
 ## Supported versions
 
-Elixir:
-
-- 1.6.0 minimum
```
Comment (balduncle) [inline README.md]: Since the version mentioned in elixir-ls is higher.

**Comment by balduncle** [comment]
@axelson per our earlier discussion on maintaining this repo I'm going to merge this.  If you or anyone else want to be involved with PR reviews please let me know.

**Comment by axelson** [comment]
Looks like a good start!

**Pull request merged**